src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ActualPlanDataReaderNodeBuilder.cs

```diff
@@ -0,0 +1,87 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.ShowPlan
```

Comment:
I'd recommend to try to split the "ShowPlanService" and the showplan graph build into separate components, and put the generic showplan code in a subfolder and the STS specific code in a code level folder.  This would allow the more generic code to be moved somewhere else in the future if we want to go that route.